src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    Event(String userInput, int startIdx) {
+        int fromIndex = userInput.indexOf(" /from ");
```

Comment:
Code quality is good and the variables are named appropropriately. Indentation is all good.